docs/source/publish_and_share_kedro_viz_on_gcp.md

```diff
@@ -0,0 +1,115 @@
+# Publish and share via GCP
+
+This page describes how to publish Kedro-Viz to Azure to share it with others. It uses the spaceflights tutorial as an example.
+
+## Setup your kedro project 
+
+If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.1). To check the version installed, type `kedro -V` in your terminal window.
+```
+
+In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project. Install the dependencies from the project root directory by typing the following in your terminal:
+
+```bash
+pip install -r requirements.txt
+```
+
+Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file of the Kedro project to add the following:
+
+```text
+fsspec[s3]>=2023.9.0
+kedro>=0.18.2
+```
+
+## Install cloud dependencies
+
+```bash
+pip install 'kedro-viz[gcp]'
+```
+
+## Configure your Google Cloud Storage
+
+You can host your Kedro-Viz project on Google Cloud Storage (GCS) bucket.
+
+1. Enable static website hosting: Follow the [GCP tutorial](https://cloud.google.com/storage/docs/hosting-static-website) to configure static website hosting on GCS.
+
+```{note}
+The process of uploading your site's files will be done through Kedro-Viz.
+```
+
+2. Ensure the `Compute Engine API` is enabled for your project as mentioned in the tutorial.
+
+3. Set up load balancer and SSL certificate: If serving your website through HTTPS, [set up a load balancer](https://cloud.google.com/storage/docs/hosting-static-website#lb-ssl) and configure an SSL certificate. 
+
+4. Set environment variable: For file uploads through Kedro-Viz, set `GOOGLE_APPLICATION_CREDENTIALS` as an environment variable. Follow these steps:
```

Comment:
This section can be made similar to Azure. Since we are not setting env variable here, we can say **Obtain Application Credentials** or something similar to Azure where we say **Obtain parameters**. We can remove the statement - For file uploads through Kedro-Viz, set GOOGLE_APPLICATION_CREDENTIALS as an environment variable

Step4 can be broken down into - Create Service account and assign a role, Generate Service Account Key and download it